src/snowflake/snowpark/session.py

```diff
@@ -3399,7 +3399,7 @@ def flatten(
         set_api_call_source(df, "Session.flatten")
         return df
 
-    def query_history(self) -> QueryHistory:
+    def query_history(self, include_describe: bool = False) -> QueryHistory:
```

Comment:
Need to update the docstring for the new parameter `include_describe`.
Pls also list this as a new feature in the changelog.